### ISSUE TITLE
Fonts antialiasing & popup styles are messed up after update to 1.7

### ISSUE THREAD

**Issue by and3rson** [dunst-project/dunst#987]
### Installation info

- Version: `Dunst - A customizable and lightweight notification-daemon 1.6.1 (2021-02-21)` && `Dunst - A customizable and lightweight notification-daemon 1.7.1 (2021-11-01)`
- Install type: `package`
- Distro and version: `ArchLinux `

### Details

Old look:

![image](https://user-images.githubusercontent.com/190107/143855564-293ccf7b-b3b7-4a8b-8816-d7821daabbae.png)

New look:

![image](https://user-images.githubusercontent.com/190107/143855583-270e3292-f74b-475d-8909-19d6d9b963de.png)

As you can see, weird white border has appeared, antialiasing has gone and the popup became more narrow.
Any info why did this happen?

My configuration: http://ix.io/3Gsl

**Comment by fwsmit** [comment]
You are using some features that have long been deprecated. The frame section has been removed in 1.7.0. Take a look at the output of dunst to see if you're using other removed features

**Comment by fwsmit** [comment]
> @fwsmit thanks, yeah, I just saw that too. Any thoughts on antialiasing?
> 
> Using default config still results in font aliasing.

I didn't change much about the font rendering, so my guess is that something else is messed up (like your font). We render fonts using pango.
Only if downgrading dunst to v1.6.1 fixes it, then it's a dunst issue.

**Comment by and3rson** [comment]
@fwsmit yes, I can confirm that downgrading to 1.6.1 fixes this as seen in the screenshots above.

**Comment by fwsmit** [comment]
> @fwsmit yes, I can confirm that downgrading to 1.6.1 fixes this as seen in the screenshots above.
> 
> Anything I can do to help you debug this?
> 
> P.S. I've ran `ldd` on two binaries to check for possible library discrepancies and the only difference is `libwayland-cursor.so.0` present in dunst `1.7.1`.

Did you reprocude it with the default config? If not, can you try that. It will make debugging it a lot easier

**Comment by and3rson** [comment]
> > @fwsmit yes, I can confirm that downgrading to 1.6.1 fixes this as seen in the screenshots above.
> > Anything I can do to help you debug this?
> > P.S. I've ran `ldd` on two binaries to check for possible library discrepancies and the only difference is `libwayland-cursor.so.0` present in dunst `1.7.1`.
> 
> Did you reprocude it with the default config? If not, can you try that. It will make debugging it a lot easier

Yes, I was able to reproduce that with the default config. I'm using Sway on ArchLinux with all packages up-to-date.

Also I've notices that SVG icons are aliased as well.

**Comment by fwsmit** [comment]
Did you configure the scale of your display to something different than 1?

**Comment by and3rson** [comment]
I only have `QT_SCALE_FACTOR=1.3`, no other scaling for current monitor:

```
    output * bg /home/anderson/.wallpapers/lights2.jpg fill
    output eDP-1 position 0,0  # primary display
    output HDMI-A-1 position 1920,0 transform 90 scale 3  # touchpad display
```

**Comment by and3rson** [comment]
@fwsmit Weird, when I reset secondary display scale to 1, the text becomes antialiased!

**Comment by and3rson** [comment]
Weird, the second display displays the popup nice and antialiased. However the primary monitor still has issue with aliasing.

**Comment by fwsmit** [comment]
> I only have `QT_SCALE_FACTOR=1.3`

This one should be ignored by dunst.
:
> 
> ```
>     output * bg /home/anderson/.wallpapers/lights2.jpg fill
>     output eDP-1 position 0,0  # primary display
>     output HDMI-A-1 position 1920,0 transform 90 scale 3  # touchpad display
> ```

The only scaling dunst takes into account is scaling via the wayland protocol, e.g. setting the scale in sway as above.


> @fwsmit Weird, when I reset secondary display scale to 1, the text becomes antialiased!
> 
> TL;DR: I'm not using 2nd display, however notifications of primary display become aliased if secondary display is scaled to 3.

That's weird indeed. Let me see if I can reproduce.

**Comment by fwsmit** [comment]
I cannot see any kind of font aliasing. Setting `follow=keyboard/mouse` will get dunst confused about the current scale, though. This is because dunst doesn't know on which monitor the window will land (this can be fixed in a kind of hacky way). Try setting `follow=none` and if needed `monitor=1` to see if that's the cause.

**Comment by fwsmit** [comment]
I did rework code that decides what scale to use in wayland recently, so that may have messed things up. But it's weird that only the text quality changes.
The window width setting has also been changed recently, see `width` in the man page.

**Comment by fwsmit** [comment]
@and3rson have you been able to solve this problem?

**Comment by bynect** [comment]
stale
